app/assets/javascripts/components/PipelineSampleReport.jsx

```diff
@@ -189,18 +202,22 @@ class PipelineSampleReport extends React.Component {
 
                         <td>{ (!taxon.genus_nt_ele) ? '-': taxon.genus_nt_ele.zscore.toFixed(3) }</td>
                         <td>{ (!taxon.genus_nt_ele) ? '-': taxon.genus_nt_ele.rpm.toFixed(3) }</td>
-                        <td>{ (!taxon.genus_nt_ele) ? '-': taxon.genus_nt_ele.count }</td>
+
                         <td>{ (!taxon.genus_nr_ele) ? '-': taxon.genus_nr_ele.zscore.toFixed(3) }</td>
                         <td>{ (!taxon.genus_nr_ele) ? '-': taxon.genus_nr_ele.rpm.toFixed(3)}</td>
-                        <td>{ (!taxon.genus_nr_ele) ? '-': taxon.genus_nr_ele.count}</td>
 
                         {/*The species scores*/}
-                        <td>{ (this.view_level==='species' && taxon.nt_ele) ? taxon.nt_ele.zscore.toFixed(3) : '' } </td>
-                        <td>{ (this.view_level==='species' && taxon.nt_ele) ? taxon.nt_ele.rpm.toFixed(3) : '' }</td>
-                        <td>{ (this.view_level==='species' && taxon.nt_ele) ? taxon.nt_ele.count : '' }</td>
-                        <td>{ (this.view_level==='species' && taxon.nr_ele) ? taxon.nr_ele.zscore.toFixed(3) : '' }</td>
-                        <td>{ (this.view_level==='species' && taxon.nr_ele) ? taxon.nr_ele.rpm.toFixed(3) : '' }</td>
-                        <td>{ (this.view_level==='species' && taxon.nr_ele) ? taxon.nr_ele.count : '' }</td>
+ 
```

Comment:
This change means that the value 0 will not be displayed. But it should be. Please change your lines 211-220 to lines 198-203 I had before. (Also including the ones that correspond to raw counts.)